pkg/controller/podgc/gc_controller.go

```diff
@@ -238,7 +249,13 @@ func (gcc *PodGCController) gcOrphaned(ctx context.Context, pods []*v1.Pod, node
 			continue
 		}
 		klog.V(2).InfoS("Found orphaned Pod assigned to the Node, deleting.", "pod", klog.KObj(pod), "node", pod.Spec.NodeName)
-		if err := gcc.deletePod(ctx, pod.Namespace, pod.Name); err != nil {
+		condition := &v1.PodCondition{
+			Type:    targetedForDisruption,
+			Status:  v1.ConditionTrue,
+			Reason:  "DeletionByPodGC",
+			Message: "To be deleted by pod garbage-collector as the node no longer exists",
```

Comment:
"podgc: node no longer exists"

Note that this is not the regular garbage collector and the current message will be confusing.

Reply:
indeed, updated, just capitalized the first letter